admin_manual/installation/server_tuning.rst

```diff
@@ -2,12 +2,6 @@
 Server tuning
 =============
 
-Using cron to perform background jobs
```

Comment:
Keep it here and just link to it from more places?

Reply:
At this point it's not just tuning, I would say that it is a requirement to run time sensitive background jobs regularly but yeah we can keep it here too

The "Installation on Linux" page is likely the most visited page but I don't have the metrics, where else would be appropriate?

Reply:
I would just have it in both. It is still a tuning for existing installations that did not do it on setup